hyperspy/tests/io/test_bruker.py

```diff
@@ -205,3 +207,16 @@ def test_decimal_regex():
         assert b'85.658' in fix_dec_patterns.sub(b'\\1.\\2', i)
     for j in dummy_xml_negative:
         assert b'.' not in fix_dec_patterns.sub(b'\\1.\\2', j)
+
+def test_all_spx_loads():
+    for spxfile in spx_files:
+        filename = os.path.join(my_path, 'bruker_data', spxfile)
+        s = load(filename)
+        assert s.data.dtype == np.uint64
+        assert s.metadata.Signal.signal_type == 'EDS_SEM'
+
+def test_stand_alone_spx():
```

Comment:
Check the value of some more metadata? tilt alpha, beam energy, data, time, etc.

Reply:
there is no need to do that as it is already extensively tested with bcf readers (which uses exactly the same code for metadata loading/transfering into hyperspy metadata dict from xml). this test tests only features which are always present in spx but not in bcf.